server/storage/storage_test.go

```diff
@@ -279,3 +282,132 @@ func TestLoadRegionsExceedRangeLimit(t *testing.T) {
 	}
 	re.NoError(failpoint.Disable("github.com/tikv/pd/server/storage/kv/withRangeLimit"))
 }
+
+const (
+	keyChars = "abcdefghijklmnopqrstuvwxyz"
+	keyLen   = 10
+)
+
+func generateKeys(size int) []string {
+	m := make(map[string]struct{}, size)
+	for len(m) < size {
+		k := make([]byte, keyLen)
+		for i := range k {
+			k[i] = keyChars[rand.Intn(len(keyChars))]
+		}
+		m[string(k)] = struct{}{}
+	}
+
+	v := make([]string, 0, size)
+	for k := range m {
+		v = append(v, k)
+	}
+	sort.Strings(v)
+	return v
+}
+
+func randomMerge(regions []*metapb.Region, n int) {
+	rand.Seed(6)
```

Comment:
Why do we use 6 here? What about using `UnixNano` as the seed?

Reply:
@HunDunDM suggests every time we run "BenchmarkLoadRegionsByRandomMerge" will create the same merge rule, so I pick up a number(6) as the seed.